src/components/infoPanels/ContactPanel.js

```diff
@@ -1,19 +1,93 @@
 import React from 'react';
-import Typography from '@material-ui/core/Typography';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Divider from '@material-ui/core/Divider';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Avatar from '@material-ui/core/Avatar';
+import ListItemText from '@material-ui/core/ListItemText';
+
+import BusinessCenterIcon from '@material-ui/icons/BusinessCenter';
+import SchoolIcon from '@material-ui/icons/School';
+import RoomServiceIcon from '@material-ui/icons/RoomService';
+import InboxIcon from '@material-ui/icons/Inbox';
+import BuildIcon from '@material-ui/icons/Build';
+import StarIcon from '@material-ui/icons/Star';
+
+import green from '@material-ui/core/colors/green';
+import amber from '@material-ui/core/colors/amber';
+import red from '@material-ui/core/colors/red';
+import purple from '@material-ui/core/colors/purple';
+import cyan from '@material-ui/core/colors/cyan';
```

Comment:
kunnen deze colors niet allemaal tegelijk geïmporteerd worden?

Reply:
https://stackoverflow.com/questions/29722270/is-it-possible-to-import-modules-from-all-files-in-a-directory-using-a-wildcard dit is een ding, maar blijkbaar is het toch niet meteen de bedoeling

Reply:
aangezien tree shaking momenteel niet goed geimplemeteerd is, wordt er in 1 optie aangeraden om ofwel alle dependencies uit een directory te importen
zie https://material-ui.com/guides/minimizing-bundle-size/